docs/articles_en/documentation/openvino_legacy_features.rst

```diff
@@ -61,25 +61,6 @@ offering.
 |   `Check the OMZ GitHub project <https://github.com/openvinotoolkit/open_model_zoo>`__
 
 
-| **Apache MXNet, Caffe, and Kaldi model formats**
-|   *New solution:* conversion to ONNX via external tools
-|   *Old solution:* model support discontinued with OpenVINO 2024.0
-|
-|   `See the previous conversion instructions <https://docs.openvino.ai/2023.3/mxnet_caffe_kaldi.html>`__
-|   :doc:`See the currently supported frameworks <openvino_docs_model_processing_introduction>`
-
-
-| **Post-training Optimization Tool (POT)**
-|   *New solution:* NNCF extended in OpenVINO 2023.0
-|   *Old solution:* POT discontinuation planned for 2024.0
-|
-|   Neural Network Compression Framework (NNCF) now offers the same functionality as POT,
-    apart from its original feature set. It is currently the default tool for performing
-    both, post-training and quantization optimizations, while POT is considered deprecated.
-|   :doc:`See how to use NNCF for model optimization <openvino_docs_model_optimization_guide>`
-|   `Check the NNCF GitHub project, including documentation <https://github.com/openvinotoolkit/nncf>`__
-
-
```

Comment:
we could keep this as an archival info. Just reduce the POT description to the bare minimum. Like the formats - 3 lines + 2 links or so.

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/ef18601b07d035c40555cb80978cca7d3fb76669